scripts/lighthouse_checks.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Lighthouse checks and store reports script."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+import subprocess
+import sys
+import time
+import python_utils
+
+from . import common
+from . import install_third_party_libs
+from . import setup
+from . import setup_gae
+
+OPPIA_SERVER_PORT = 8181
+COMPY_SERVER_PORT = 9999
+RUNNING_PROCESSES = []
+GO_VERSION = '1.14.1'
+GO_PATH = os.path.join(common.OPPIA_TOOLS_DIR, 'go-%s' % GO_VERSION)
+GO_BINARY = os.path.join(GO_PATH, 'bin', 'go')
+GO_HOME_PATH = os.path.join(os.path.expanduser('~'), 'go')
+
+COMPY_BINARY = os.path.join(GO_HOME_PATH, 'bin', 'compy')
+COMPY_DOWNLOAD_PATH = os.path.join(
+    GO_HOME_PATH, 'src', 'github.com', 'barnacs', 'compy')
+MAX_WAIT_TIME_FOR_PORT_TO_OPEN_SECS = 1000
+
+
+def setup_and_install_dependencies():
+    """Run the setup and installation scripts."""
+
+    install_third_party_libs.main()
```

Comment:
I think it is necessary just to ensure they have it
 @jameesjohn Hey you could you take a look at this?

Reply:
It could be removed actually. 
I added it initially so that we could use it in the GitHub action, but we could run the third party script independently of this.